x-pack/plugins/monitoring/public/components/chart/monitoring_timeseries_container.js

```diff
@@ -18,19 +19,31 @@ export function MonitoringTimeseriesContainer({ series, onBrush }) {
     return null; // still loading
   }
 
+  const title = getTitle(series);
+  const titleForAriaIds = title.replace(/\s+/, '--');
   const units = getUnits(series);
+  const bucketSize = get(first(series), 'bucket_size'); // bucket size will be the same for all metrics in all series
+
+  const seriesScreenReaderTextList = [`Interval: ${bucketSize}`];
+  seriesScreenReaderTextList.push(...series.map(item => `${item.metric.label}: ${item.metric.description}`));
 
   return (
     <div className="monitoring-chart__container">
-      <h2 className="euiTitle">
-        { getTitle(series) }{ units ? ` (${units})` : '' }
+      <h2 className="euiTitle" tabIndex="0">
+        <span className="kuiScreenReaderOnly">This chart is not accessible yet</span>
+        { title }{ units ? ` (${units})` : '' }
         <OverlayTrigger
           placement="left"
           trigger="click"
-          overlay={<Tooltip><InfoTooltip series={series}/></Tooltip>}
+          overlay={<Tooltip><InfoTooltip series={series} bucketSize={bucketSize}/></Tooltip>}
         >
           <span className="monitoring-chart-tooltip__trigger overlay-trigger">
-            <KuiInfoButton />
+            <Fragment>
+              <KuiInfoButton aria-labelledby={`monitoringChart${titleForAriaIds}`} />
+              <span id={`monitoringChart${titleForAriaIds}`} className="kuiScreenReaderOnly">
```

Comment:
I added this as a way for the screen reader to connect the content in the tooltip with the button showing it. I couldn't do this in the `InfoTooltip` itself as that doesn't render any DOM until the button is clicked. Perhaps there is a better way?